src/goto-programs/goto_program.cpp

```diff
@@ -604,7 +621,6 @@ void goto_programt::copy_from(const goto_programt &src)
   compute_target_numbers();
 }
 
-// number them
 bool goto_programt::has_assertion() const
```

Comment:
Would you mind adding documentation (something along the lines of "Returns true if, and only if, the goto program includes an ASSERT instruction the guard of which is not trivially true.")